docs/aro/setup-quay/quay-console.md

```diff
@@ -0,0 +1,316 @@
+
+    ```bash
+    AZR_RESOURCE_LOCATION=eastus
+    AZR_RESOURCE_GROUP=openshift
+    AZR_CLUSTER=cluster
+    AZR_PULL_SECRET=~/Downloads/pull-secret.txt
+    ```
+
+1. Create an Azure resource group
+
+    ```bash
+    az group create \
+      --name $AZR_RESOURCE_GROUP \
+      --location $AZR_RESOURCE_LOCATION
+    ```
+
+
+### Networking
+
+Create a virtual network with two empty subnets
+
+1. Create virtual network
+
+    ```bash
+    az network vnet create \
+      --address-prefixes 10.0.0.0/22 \
+      --name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" \
+      --resource-group $AZR_RESOURCE_GROUP
+    ```
+
+1. Create control plane subnet
+
+    ```bash
+    az network vnet subnet create \
+      --resource-group $AZR_RESOURCE_GROUP \
+      --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" \
+      --name "$AZR_CLUSTER-aro-control-subnet-$AZR_RESOURCE_LOCATION" \
+      --address-prefixes 10.0.0.0/23 \
+      --service-endpoints Microsoft.ContainerRegistry
+    ```
+
+1. Create machine subnet
+
+    ```bash
+    az network vnet subnet create \
+      --resource-group $AZR_RESOURCE_GROUP \
+      --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" \
+      --name "$AZR_CLUSTER-aro-machine-subnet-$AZR_RESOURCE_LOCATION" \
+      --address-prefixes 10.0.2.0/23 \
+      --service-endpoints Microsoft.ContainerRegistry
+    ```
+
+1. Disable network policies on the control plane subnet
+
+    > This is required for the service to be able to connect to and manage the cluster.
+
+    ```bash
+    az network vnet subnet update \
+      --name "$AZR_CLUSTER-aro-control-subnet-$AZR_RESOURCE_LOCATION" \
+      --resource-group $AZR_RESOURCE_GROUP \
+      --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" \
+      --disable-private-link-service-network-policies true
+    ```
+
+1. Create the cluster
+
+    > This will take between 30 and 45 minutes.
+
+    ```bash
+    az aro create \
+      --resource-group $AZR_RESOURCE_GROUP \
+      --name $AZR_CLUSTER \
+      --vnet "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" \
+      --master-subnet "$AZR_CLUSTER-aro-control-subnet-$AZR_RESOURCE_LOCATION" \
+      --worker-subnet "$AZR_CLUSTER-aro-machine-subnet-$AZR_RESOURCE_LOCATION" \
+      --worker-vm-size Standard_D16s_v3 \
+      --pull-secret @$AZR_PULL_SECRET
+    ```
+
+1. Get OpenShift console URL
+
+    ```bash
+    az aro show \
+      --name $AZR_CLUSTER \
+      --resource-group $AZR_RESOURCE_GROUP \
+      -o tsv --query consoleProfile
+    ```
+
+1. Get OpenShift credentials
+
+    ```bash
+    az aro list-credentials \
+      --name $AZR_CLUSTER \
+      --resource-group $AZR_RESOURCE_GROUP \
+      -o tsv
+    ```
+
+1. Use the URL and the credentials provided by the output of the last two commands to log into OpenShift via a web browser.
+
+![ARO login page](./images/aro-login.png)
+
+## Red Hat Quay Setup
+
+### Backend Storage Setup
+
+1. **Login** to Azure
```

Comment:
provide a link to portal.azure.com or whatever the base URL is for azure cloud console..